tests/cli/main.cpp

```diff
@@ -197,102 +199,118 @@ class client_connection
 };
 
 ///////////////////////////////
-// Tests
+// Cli Wallet Fixture
 ///////////////////////////////
 
-////////////////
-// Start a server and connect using the same calls as the CLI
-////////////////
-BOOST_AUTO_TEST_CASE( cli_connect )
+struct cli_fixture
 {
-   using namespace graphene::chain;
-   using namespace graphene::app;
+   int server_port_number;
+   fc::temp_directory app_dir;
    std::shared_ptr<graphene::app::application> app1;
-   try {
-      fc::temp_directory app_dir ( graphene::utilities::temp_directory_path() );
+   client_connection con;
+   std::vector<std::string> nathan_keys;
+
+   cli_fixture() : 
+   server_port_number(0),
```

Comment:
I think we can use some indentations here.

Reply:
ok sure !